docs/concepts/work-pools.md

```diff
@@ -430,6 +430,9 @@ Priority for flow run submission proceeds from the highest priority to the lowes
 
 If new flow runs are received on the "critical" queue while flow runs are still in scheduled on the "high" and "low" queues, flow run submission goes back to ensuring all scheduled work is first satisfied from the highest priority queue, until it is empty, in waterfall fashion.
 
+!!! tip "Work queue status"
+    A work queue has a `READY` status when it has been polled by a worker in the last 60 seconds. Pausing a work queue will give it a `PAUSED` status, meaning no work will be added to that work queue. Unpausing a work queue will lead to a `NOT_READY` status unless a worker has polled it in the last 60 seconds.
```

Comment:
```suggestion
    A work queue has a `READY` status when it has been polled by a worker in the last 60 seconds. Pausing a work queue will give it a `PAUSED` status and mean that it will accept no new work until it is unpaused. A user can control the work queue's paused status in the UI. Unpausing a work queue will give the work queue a `NOT_READY` status unless a worker has polled it in the last 60 seconds.
```